src/icons/tritcher.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
```

Comment:
Did you mean "Trichter"? 

https://www.conrad.de/de/p/pressol-02363-trichter-0-25-l-100-mm-823648.html

Reply:
hehehe thanks buddy, is fixed